src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudServiceActor.scala

```diff
@@ -74,7 +82,7 @@ class FireCloudServiceActor extends HttpServiceActor with FireCloudDirectives {
         userService.routes ~
         nihSyncService.routes ~
         healthService.routes ~
-        libraryService.routes ~
+        libraryRoutes ~
```

Comment:
first, it's part of aligning rawls/orchestration. But more importantly, `val libraryApiService = new LibraryApiService with ActorRefFactoryContext` doesn't work, because it can't specify a `libraryServiceConstructor`.